org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeChannelConverterFactory.java

```diff
@@ -92,7 +92,6 @@ public ZigBeeChannelConverterFactory() {
      *
      * @param thingUID the {@link ThingUID} of the thing
      * @param endpoint the {@link ZigBeeEndpoint} to generate the channels for
-     * @return
```

Comment:
Wouldn't it be better to add the Javadoc return type here rather than delete the reference?